packages/datetime/test/dateRangePickerTests.tsx

```diff
@@ -402,6 +402,36 @@ describe("<DateRangePicker>", () => {
     });
 
     describe("hover interactions", () => {
+        describe("react-day-picker props are used", () => {
+            it("calls onMonthChange", () => {
+                const monthChangeSpy = sinon.spy();
```

Comment:
you can just call this `onMonthChange`. it's an emerging convention in the codebase. makes this line nice and succinct:

```tsx
renderDateRangePicker({ dayPickerProps: { onMonthChange } });
```